src/UpdateTracker.js

```diff
@@ -41,14 +41,17 @@ export default class UpdateTracker {
   unsubscribe(...urls) {
     for (let url of urls) {
       url = url.replace(/#.*/, '');
-      if (url in subscribers)
+      if (url in subscribers) {
         subscribers[url].delete(this.subscriber);
+        delete subscribers[url];
+        delete webSockets[new URL(url).host];
```

Comment:
This seems to cause issues when an `UpdateTracker` is subscribed to two URLs from the same host, and we are only unsubscribing from one.

Reply:
Yeah I agree this was incorrect and I've reverted it.